tests/tensorflow/test_tensorflow_autolog.py

```diff
@@ -0,0 +1,213 @@
+# pep8: disable=E501
+
+from __future__ import print_function
+
+import collections
+import shutil
+import pytest
+import tempfile
+
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+from tensorflow.python.keras import layers
+
+import mlflow
+import mlflow.tensorflow
+
+SavedModelInfo = collections.namedtuple(
+        "SavedModelInfo",
+        ["path", "meta_graph_tags", "signature_def_key", "inference_df", "expected_results_df"])
+
+
+@pytest.fixture
+def tf_keras_random_data():
```

Comment:
Nit: maybe rename this to `tf_keras_random_data_run_id` to make it clearer the fixture returns a run ID?

Reply:
You could actually just make this `tf_keras_random_data_run` and just return the run (instead of the ID), since it seems the ID is only ever used to query for the run.